int16 uav_id
time stamp
Candidate[] candidates
